dss/util/auth/authregistry.py

```diff
@@ -0,0 +1,15 @@
+import typing
+
+
+class AuthRegistry(type):
+
+    REGISTRY: typing.Dict[typing.Any, typing.Any] = {}
+
+    def __new__(cls, name, bases, attrs):
+        new_cls = type.__new__(cls, name, bases, attrs)
+        cls.REGISTRY[new_cls.__name__] = new_cls
```

Comment:
```suggestion
        cls.REGISTRY[new_cls.__name__.lower()] = new_cls
```

This avoids confusion around whether the registry is case-sensitive, etc. See associated change in `AuthHandler`

Reply:
:gir

Reply:
thanks! i added it in